packages/common/src/objects/deepFilter.ts

```diff
@@ -71,8 +71,15 @@ export function deepFilter(
     return Object.keys(object).reduce((acc, entry) => {
       if (predicate(object[entry])) {
         if (isFindable(object[entry])) {
-          const filteredEntry = deepFilter(object[entry], predicate);
-          (acc as any)[entry] = filteredEntry;
+          // Deep filter will run through a Blob and clear out the Blob specific properties
```

Comment:
Maybe change this comment to be less about what it did before, and more what it does not... something like...
`// Explicilty checking for Blob here, and copying the reference forward so it is maintained`